scout/constants/clinvar.py

```diff
@@ -1,4 +1,4 @@
-CLINVAR_API_URL = "https://submit.ncbi.nlm.nih.gov/api/v1/submissions/"
```

Comment:
This change is for the sake of testing only, it should be reverted immediately before merging this PR!